client/src/components/ProfilePublic/ProfilePublic.styled.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import { lighten } from 'polished';
+
+import { $grey, $violet } from '../../assets/styledVars';
+
+import { UserName, Dashboard } from '../Profile/Profile.styled';
+
+export const ProfileSection = styled.section`
+	display: grid;
+	width: 100%;
+	height: 75vh;
+	grid-template-columns: 1fr;
+	grid-template-rows: minmax(45%, min-content) 1fr;
+`;
+
+export const Header = styled.div`
+	background-color: ${lighten(0.5, $grey)};
+	border-radius: 6px 0 0 6px;
```

Comment:
I'm not sure if you meant this to be vertically stacked, different from the logged in user's profile layout.

Reply:
I guess a benefit in this is differentiating it from the user's personal profile, although this leaves a lot of space on either side here. If it is to be vertically stacked this border radius should be `6px 6px 0 0`.

Reply:
I started with the current Profile layout but there was just too much empty space and not that much publicly available info about users, so I've reused some styles from Profile and created another component. This will need some changes for sure.